web/src/components/Chip/Chip.scss

```diff
@@ -9,3 +9,33 @@
   margin: 0.25rem;
   @include general.highlight-hover;
 }
+
+
+@keyframes shakeBackground {
+  // Do not edit the steps of this animation unless you intend to not use box-shadow
```

Comment:
You may want to update this comment with a message about why we want to shake the background. Something like "Get the users attention but don't be obnoxious about it. Used For things that aren't errors, but need some user attention like a source with no zones "